src/main/java/net/fabricmc/loom/configuration/mods/ArtifactMetadata.java

```diff
@@ -58,11 +61,20 @@ public static ArtifactMetadata create(ArtifactRef artifact) throws IOException {
 			if (Files.exists(manifestPath)) {
 				final var manifest = new Manifest(new ByteArrayInputStream(Files.readAllBytes(manifestPath)));
 				final Attributes mainAttributes = manifest.getMainAttributes();
-				final String value = mainAttributes.getValue(MANIFEST_REMAP_KEY);
+				final String remapKey = mainAttributes.getValue(MANIFEST_REMAP_KEY);
```

Comment:
Nit: this isn't really a "remap key", I'm pretty sure the `_KEY` in the attribute field name is "key" as in a map key in the attribute map